src/com/rockbite/bootcamp/store/Shop/Shop.java

```diff
@@ -0,0 +1,154 @@
+package com.rockbite.bootcamp.store.Shop;
+
+import com.rockbite.bootcamp.store.Command.CommandManager;
+import com.rockbite.bootcamp.store.IInventory;
+import com.rockbite.bootcamp.store.Pool.IPool;
+import com.rockbite.bootcamp.store.Pool.Pool;
+import com.rockbite.bootcamp.store.Product;
+import com.rockbite.bootcamp.store.User;
+import com.rockbite.bootcamp.store.collections.Inventory.Inventory;
+import com.rockbite.bootcamp.store.collections.Resources.Item;
+
+public class Shop implements IShop, IPool {
+    /**
+     * identification for Shop
+     */
+    private int id;
+    /**
+     * list of products in shop with their prices
+     */
+    private Inventory productList = new Inventory();
+
+
+    /**
+     * operation in shop count
+     */
+    private Integer operationCounter = 0;
+    private Integer reOperationCounter = 0;
+    /**
+     * shop manager
+     */
+    public CommandManager manager = new CommandManager();
+    public Pool<IncrementCommand> incrementCommandPool = new Pool<IncrementCommand>() {
+        @Override
+        protected IncrementCommand newObject() {
+            return new IncrementCommand();
+        }
+    };
+    public Pool<DecrementCommand> decrementCommandPool = new Pool<DecrementCommand>() {
+        @Override
+        protected DecrementCommand newObject() {
+            return new DecrementCommand();
+        }
+    };
+
+
+
+
+
+
+    /**
+     * adding products to shop storage
+     * @param product
+     * @param count
+     */
+    public void addProduct(Product product, int count){
+        productList.addItem(product, count);
+    }
+
+    /**
+     * remove products frome storage
+     * @param product
+     * @param count
+     */
+    public void removeProduct(Product product, int count){
+        productList.removeItem(product, count);
+    }
+
+    /**
+     * get all products from storage
+     * @return
+     */
+    public Inventory getProductList() {
+        return productList;
+    }
+
+    @Override
+    public void getProduct(Product product, int count) {
+
+    }
+
+    @Override
+    public boolean canAfford(IInventory user, Product product) {
+        if (productList.getCollection().get(product)!=null){
```

Comment:
Inconsistent spacing around control structures. 

if (){ in the first line, 
for (...) { in the second line